### PR TITLE
components - implement share modal with member list in project

### PR DESCRIPTION
**Pull request by junhoyeo**
localhost_3000_screens

I changed the wrong typings on the other files as well 🚀

### CODE REVIEW

File: web/components/canvas/selectable-layer.tsx [hunk 1]
```diff
@@ -1,60 +1,53 @@
...
+interface ISelectableLayer {
   children: JSX.Element;
   width: number;
   height: number;
   selected?: boolean;
   onFocusChange?: (id: string, focus: boolean) => void;
-}) {
+}
+
+export const SelectableLayer: React.FC<ISelectableLayer> = ({
```
Comment (softmarshmallow) [inline web/components/canvas/selectable-layer.tsx:12]: What is the reason changing functions as to arrow functions?

Comment (junhoyeo) [reply web/components/canvas/selectable-layer.tsx:12]: This is more intuitive - I wanted to reduce code duplications by passing size params once with `size`, an object with `width` & `height`. we can structurize it

**Comment by junhoyeo** [comment]
Updated placeholder text

File: web/components/canvas/selectable-layer.tsx (hunk 1 above)
Comment (softmarshmallow) [reply web/components/canvas/selectable-layer.tsx:12]: Well second thoughts on this one. I'll put this one too on the agenda. Merging 😎

**Pull request merged**